packages/nc-gui-v2/components/cell/Url.vue

```diff
@@ -35,14 +41,37 @@ const url = computed(() => {
 })
 
 const focus: VNodeRef = (el) => (el as HTMLInputElement)?.focus()
+
+watch(
+  () => editEnabled.value,
+  () => {
+    if (column.value.meta?.validate && !editEnabled.value && localState.value && !isValidURL(localState.value)) {
+      message.error('Invalid URL')
+      localState.value = undefined
+      return
+    }
+    localState.value = value
+  },
+)
 </script>
 
 <template>
-  <input v-if="editEnabled" :ref="focus" v-model="vModel" class="outline-none text-sm" @blur="editEnabled = false" />
+  <div class="flex flex-row items-center justify-between">
+    <input v-if="editEnabled" :ref="focus" v-model="vModel" class="outline-none text-sm w-full" @blur="editEnabled = false" />
+
+    <nuxt-link v-else-if="isValid" class="text-sm underline hover:opacity-75" :to="url" target="_blank">{{ value }} </nuxt-link>
 
-  <nuxt-link v-else-if="isValid" class="text-sm underline hover:opacity-75" :to="url" target="_blank">{{ value }} </nuxt-link>
+    <span v-else class="w-9/10 overflow-ellipsis overflow-hidden">{{ value }}</span>
 
-  <span v-else>{{ value }}</span>
+    <div v-if="!isValid && value?.length && !editEnabled" class="mr-1 w-1/10">
+      <a-tooltip placement="top">
+        <template #title> Invalid URL </template>
+        <div class="flex flex-row items-center">
+          <MaterialSymbolsWarning />
```

Comment:
Suggest to color it.

![image](https://user-images.githubusercontent.com/35857179/185288545-71c2d932-0b5f-4792-8684-875a8fc792d5.png)